client/me/purchases/cancel-purchase/loading-placeholder.jsx

```diff
@@ -1,33 +1,94 @@
-import { Button, Card, CompactCard } from '@automattic/components';
-import { localize } from 'i18n-calypso';
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate, localize } from 'i18n-calypso';
+import page from 'page';
 import PropTypes from 'prop-types';
-import LoadingPlaceholder from 'calypso/me/purchases/components/loading-placeholder';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import { CALYPSO_CONTACT } from 'calypso/lib/url/support';
 import titles from 'calypso/me/purchases/titles';
 
+import '../components/loading-placeholder/style.scss';
+
+const SupportLink = () => {
+	const translate = useTranslate();
+	return translate( 'Have a question? {{contactLink}}Ask a Happiness Engineer!{{/contactLink}}', {
+		components: {
+			contactLink: <a href={ CALYPSO_CONTACT } />,
+		},
+	} );
+};
+
+const siteHeader = () => {
```

Comment:
Minor: this could also be a component: `<SiteHeader />`

Reply:
SiteHeader is now a react component